docs/sources/developers/angular_deprecation/_index.md

```diff
@@ -14,7 +14,17 @@ weight: 500
 
 # Angular support deprecation
 
-Angular plugin support is deprecated and will be removed in a future release. There are still many community plugins that rely on Grafana's Angular plugin support to work. The same is true for many internal (private) plugins that have been developed by Grafana users over the years. Grafana version 9 has a server configuration option that is global to the entire instance and controls whether Angular plugin support is available or not. By default, Angular support is still enabled, but that will change soon once we complete the migration of all Angular code in the core product.
+Angular plugin support is deprecated and will be removed in a future release. There are legacy Core Grafana visualizations and external plugins that rely on Grafana's Angular plugin support to work. The same is true for many internal (private) plugins that have been developed by Grafana users over the years. From Grafana v9 and onwards, there is a [server configuration option](<(https://github.com/grafana/grafana/blob/d61bcdf4ca5e69489e0067c56fbe7f0bfdf84ee4/conf/defaults.ini#L362)>) that is global to the entire instance and controls whether Angular plugin support is available or not. By default, Angular support is still enabled, but that will change soon once we complete the migration of all Angular code in the core product.
```

Comment:
Sorry that I've reviewed the whole section, this is partly because GitHub's line based diffing algorithm can't give a more accurate diff.

[Semantic line breaks](https://sembr.org/) are better for future line based diffing in the GitHub UI and result in the same HTML output.

The suggestion:
- Uses semantic line breaks to improve future diffing
- Removes capitalization of "Core" because I'm not sure that it's a proper noun
- Removes "(private)" parenthetical, I'm not sure that it will be meaningful to readers.
  What would a user think a "private" plugin is?

```suggestion
Angular plugin support is deprecated and will be removed in a future release.
There are legacy core Grafana visualizations and external plugins that rely on Grafana's Angular plugin support to work.
The same is true for many internal plugins that have been developed by Grafana users over the years.
From Grafana v9 and onwards, there is a [server configuration option](https://github.com/grafana/grafana/blob/d61bcdf4ca5e69489e0067c56fbe7f0bfdf84ee4/conf/defaults.ini#L362) that's global to the entire instance and controls whether Angular plugin support is available or not.
By default, Angular support is still enabled, but that will change after the migration of all Angular code in the core product.
```